### PR TITLE
Radio button usage updates

### LINKED ISSUE #2096: [radio button] update usage tab docs
Update radio button usage with component docs template

## Overview

- [x] content
- [x] when to use/not use

## Formatting

- [x] anatomy
- [x] alignment
- [x] placement

## Content

- [x] main elements
- [x] overflow

## Behaviors

- [x] states
- [x] interaction
- [x] validation

### PR DESCRIPTION
**Pull request by kingtraceyj**
Closes #2096 

Updating usage docs to the new templates

### Changelog

**New sections added**
- Added Content
- Added Behaviors
- Added Related
- Added References

**Updated sections**
- Updated all images
- Updated Overview page—added when to use and when not to use
- Updated Formatting page—added anatomy, alignment, and placement sections

### CODE REVIEW

**Review by tw15egan — APPROVED** [review]
LGTM 👍 ✅

**Review by tay1orjones — APPROVED** [review]
Looks good to me! 🚀

**Review by janchild — COMMENTED** [review]
@kingtraceyj for the description can we change the end of the sentence to read "only one selection from the group can be made." Sorry I didn't pick that up earlier. Reviewing the rest now.

File: src/pages/components/radio-button/usage.mdx [hunk 1]
```diff
@@ -17,18 +17,61 @@ one selection from the group is allowed.
...
 </AnchorLinks>
 
 ## Overview
 
-Radio buttons represent a group of mutually exclusive choices, compared to
-checkboxes that allow users to make one or more selections from a group. In use
-cases where only one selection from a group is allowed, use the radio button
-component instead of the checkbox.
+Radio buttons are used for mutually exclusive choices, not for multiple choices.
+Only one radio button should be selected at a time. When a user chooses a new
```
Comment (janchild) [inline src/pages/components/radio-button/usage.mdx:32]:
```suggestion
Only one radio button can be selected at a time. When a user chooses a new
```

**Review by janchild — COMMENTED** [review]
@kingtraceyj This looks fantastic. I love the Related section. I just had a couple of very minor content tweaks. Otherwise it's fab!

**Review by janchild — APPROVED** [review]
🥂

**Review by jeanservaas — COMMENTED** [review]
Great job Tracey, just a couple of things!

In this image in particular, I noticed that you have two scales represented, e.g. the header bar is even smaller than that input field at the bottom of the image. So I would try to align the scale of the page content with the header so it looks more realistic as an in-context usage image.


image

You could either show the whole UI like Lauren does with checkbox and just scale the xlg design down to fit (I have a percentage for the 8 column images in that production file)

image

or just make sure you're show part of a real UI, so if any scaling is done, it's done together 

Here's an example from button usage:

image

### I'd also give these examples a little more room... 

image

I'd match the size of the canvas to this one so it can breathe a little more


image


Otherwise looks great!

**Comment by jeanservaas** [comment]
Looks great!

@kingtraceyj This is the tiniest little change:

image


Right now the input field looks like it's the same height as the header bar... which would make the input field 48px when it would actually be 40px. To get the relationship right, I'd just grab both an input field and a header bar from the kit and scale them down proportionally just to get that relationship right. 

I'm going to approve because I'll be out next week, but just that little update and it'll be perfect!!!

**Pull request merged**